acceptance/tests/environment/use_enc_environment_for_files.rb

```diff
@@ -11,24 +11,38 @@
 END
 on master, "chmod 755 #{testdir}/enc.rb"
 
-on master, "mkdir -p #{testdir}/modules"
-# Create a plugin file on the master
-on master, "mkdir -p #{testdir}/special/amod/files"
-create_remote_file(master, "#{testdir}/special/amod/files/testy", "special_environment")
-
-on master, "chown -R #{master['user']}:#{master['group']} #{testdir}"
-on master, "chmod -R g+rwX #{testdir}"
+apply_manifest_on(master, <<-MANIFEST, :catch_failures => true)
+  File {
+    ensure => directory,
+    mode => "0770",
+    owner => #{master.puppet['user']},
+    group => #{master.puppet['group']},
+  }
+  file {
+    '#{testdir}/environments':;
+    '#{testdir}/environments/production':;
+    '#{testdir}/environments/special/':;
+    '#{testdir}/environments/special/manifests':;
+    '#{testdir}/environments/special/modules':;
+    '#{testdir}/environments/special/modules/amod':;
+    '#{testdir}/environments/special/modules/amod/files':;
+  }
+  file { '#{testdir}/environments/special/modules/amod/files/testy':
+    ensure => file,
+    mode => "0640",
+    content => 'special_environment',
+  }
+MANIFEST
 
 master_opts = {
+  'main' => {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.